jwql/instrument_monitors/common_monitors/bias_monitor.py

```diff
@@ -60,6 +60,7 @@
 from jwql.utils.monitor_utils import update_monitor_table  # noqa: E402 (module import not at top)
 from jwql.utils.permissions import set_permissions  # noqa: E402 (module import not at top)
 from jwql.utils.utils import copy_files, ensure_dir_exists, filesystem_path, get_config  # noqa: E402 (module import not at top)
+from jwql.website.apps.jwql.monitor_pages.monitor_bias_bokeh import BiasMonitorPlots  # noqa: E402 (module import not at top)
```

Comment:
After merging, we'll have to create the initial copy of the instrument-specific bias pages manually. Then they will be updated each time the bias monitor runs. To create them manually, we just need to do:

from jwql.website.apps.jwql.monitor_pages.monitor_bias_bokeh import BiasMonitorPlots
for inst in ['nircam', 'niriss', 'nirspec']:
    BiasMonitorPlots('niriss')